packages/node_modules/@webex/internal-plugin-calendar/src/index.js

```diff
@@ -65,22 +65,40 @@ registerInternalPlugin('calendar', Calendar, {
             return Promise.resolve();
           }
 
-          // Decrypt participant properties
-          const decryptedParticipants = object.encryptedParticipants.map((participant) => Promise.all([
-            ctx.transform('decryptTextProp', 'encryptedEmailAddress', object.encryptionKeyUrl, participant),
-            ctx.transform('decryptTextProp', 'encryptedName', object.encryptionKeyUrl, participant)
-          ]));
+          // Check if notes and participants were sent with meeting object, and fetch if not
+          if (!object.encryptedNotes) {
+            object.encryptedNotes = Calendar.prototype.getNotes(object.id);
+          }
+          if (!object.encryptedParticipants) {
+            object.encryptedParticipants = Calendar.prototype.getParticipants(object.id);
+          }
```

Comment:
Don't fetch new data in a transform. Instead you can have the `list` command fetch participants and notes if they aren't available or make the user do a separate fetch. Add a new transform for the participant fetch and the notes fetch.

Reply:
I wrote a transform each for notes and participants, and moved the work to `.then()` of `list` fetch.

Unit tests failed but I can't see where/why?
![image](https://user-images.githubusercontent.com/50709949/74147458-48626700-4c29-11ea-8b44-510b1241d10d.png)